cmd/infra/aws/iam.go

```diff
@@ -125,6 +125,7 @@ const (
         "ec2:DetachVolume",
         "ec2:RevokeSecurityGroupIngress",
         "ec2:DescribeVpcs",
+        "ec2:DescribeDhcpOptions",
```

Comment:
this should be add to `nodePoolPolicy`, not `cloudControllerPolicy` right?

Reply:
In CAPA is included in `ControllersPolicy` https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/cmd/clusterawsadm/cloudformation/bootstrap/cluster_api_controller.go#L134 I was thinking that it belong to CCP. Let me change it.